apps/api/v2/src/modules/conferencing/outputs/get-default-conferencing-app.output.ts

```diff
@@ -14,6 +14,11 @@ export class DefaultConferencingAppsOutputDto {
   @IsOptional()
   @Expose()
   readonly appLink?: string;
+
+  @IsNumber()
+  @IsOptional()
+  @Expose()
```

Comment:
I think that the properties in `DefaultConferencingAppsOutputDto` won't show up in v2 docs because missing `ApiProperty` - we need to add it so customers know what data to expect.